jcl/src/java.base/share/classes/com/ibm/oti/vm/VM.java

```diff
@@ -299,6 +299,13 @@ public static void ensureCalledFromBootstrapClass() {
  */
 public static native void dumpString(String str);
 
+/**
+ * Native used to get a jfr option string
+ *
+ * @return a String which is jfr option
+ */
+public static native String getjfrCMDLineOption();
```

Comment:
all Java JFR code should be wrapped with `/*[IF JFR_SUPPORT]*/` and `/*[ENDIF] JFR_SUPPORT */`